protocol/consistencychecks.go

```diff
@@ -45,18 +43,48 @@ func NewCC(savedSTR *DirSTR, useTBs bool, signKey sign.PublicKey) *ConsistencyCh
 	if !useTBs {
 		panic("[coniks] Currently the server is forced to use TBs")
 	}
+	aud := NewAuditor(signKey, savedSTR)
 	cc := &ConsistencyChecks{
```

Comment:
I agree. This isn't currently consistent in all of the code, so we should fix this everywhere.